src/main/resources/assets/admin/common/js/app/wizard/WizardHeaderWithDisplayNameAndName.ts

```diff
@@ -104,9 +104,13 @@ export class WizardHeaderWithDisplayNameAndName
         this.autoGenerationEnabled = value;
     }
 
-    setDisplayName(value: string, isGeneratedValue?: boolean) {
-        this.displayNameEl.setValue(value.trim());
-        this.displayNameEl.toggleClass(WizardHeaderWithDisplayNameAndName.GENERATED_CLASS, isGeneratedValue);
+    setDisplayName(value: string, silent?: boolean) {
```

Comment:
Value of `silent` parameter is never used.

Reply:
My bad, fixed